widgets/src/components/TaskList/TaskListContainer.js

```diff
@@ -75,19 +75,27 @@ class TaskList extends React.Component {
       // dispatch onSelectTask event for the first item on list
       onSelectTask(rows[0]);
 
-      this.setState({
-        loading: false,
-        columns: normalizeColumns(JSON.parse(config.columns), rows[0], options, {
-          openDiagram: this.openDiagram,
-          selectTask: onSelectTask,
-        }),
-        rows,
-        lastPage: taskList.metadata.lastPage === 1,
-        size: taskList.metadata.size,
-        connection: config.knowledgeSource,
-      });
+      if (!taskList.payload) {
```

Comment:
this validation should be before 
`const options = JSON.parse(config.options);`
at line 71, before trying to normalize the rows.

Reply:
@gReis89 , I have moved JSON parsing and normalize rows process after `payout.length` has been checked. It would be better to check if payload contains something before working on the normalization process